src/interpreter/mod.rs

```diff
@@ -491,12 +491,12 @@ pub enum SatisfiedConstraint {
     ///Relative Timelock for CSV.
     RelativeTimelock {
         /// The value of RelativeTimelock
-        time: u32,
+        n: Sequence,
     },
     ///Absolute Timelock for CLTV.
     AbsoluteTimelock {
         /// The value of Absolute timelock
-        time: u32,
+        n: LockTime,
```

Comment:
Its a block height or a block time. `n` is from `n OP_CLTV` and `nLockTime` - that's why I used `n`.

Reply:
`n` in `nLockTime` literally means number. It's meaningless. I do see that `time` is confusing, so `lock_time` would be better because most people already know it can be height.

Reply:
Used `lock_time` for this field and similarly `sequence` for `SatisfiedConstraint::RelativeTimelock` field.